.lintstagedrc

```diff
@@ -0,0 +1,3 @@
+{
+    "*.(ts|tsx)": ["eslint --fix", "prettier --write"]
```

Comment:
@germain-gg I'm going through repos right now, and wondering if we should leave tsc typecheck for CI - it might be quite slow, what do you think about it?

Stylelint added, pushing soon-ish :)

Reply:
> This needs to happen on a lot more than just the ts and tsx files.

Yeah prettier could/should be ran on all files, its quick and guarded by `.prettierignore`

> leave tsc typecheck for CI - it might be quite slow

Agreed for the first cut